cartography/data/indexes.cypher

```diff
@@ -65,8 +65,8 @@ CREATE INDEX IF NOT EXISTS FOR (n:AWSUser) ON (n.name);
 CREATE INDEX IF NOT EXISTS FOR (n:AWSUser) ON (n.lastupdated);
 CREATE INDEX IF NOT EXISTS FOR (n:AWSVpc) ON (n.id);
 CREATE INDEX IF NOT EXISTS FOR (n:AWSVpc) ON (n.lastupdated);
-CREATE INDEX IF NOT EXISTS FOR (n:AccountAccessKey) ON (n.accesskeyid);
-CREATE INDEX IF NOT EXISTS FOR (n:AccountAccessKey) ON (n.lastupdated);
+CREATE INDEX IF NOT EXISTS FOR (n:UserAccessKey) ON (n.accesskeyid);
```

Comment:
Add an additional property `id`, which is will be equivalent to accesskeyid

Reply:
I think this might be on a previous version of the code - as it currently stands the following is the code 

```
CREATE INDEX IF NOT EXISTS FOR (n:UserAccessKey) ON (n.accesskeyid);
CREATE INDEX IF NOT EXISTS FOR (n:UserAccessKey) ON (n.lastupdated);
````

Reply:
this is what I mean, along with another suggestion below upon MERGE
```suggestion
CREATE INDEX IF NOT EXISTS FOR (n:UserAccessKey) ON (n.id);
CREATE INDEX IF NOT EXISTS FOR (n:UserAccessKey) ON (n.accesskeyid);
```